setup.py

```diff
@@ -193,24 +197,33 @@ def extract_version():
     return version
 
 
+custom_commands = {
+    'test': SetupTestRunner,
+    'develop': custom_cmd(
+        develop_cmd, [build_std_names, compile_pyke_rules]),
+    'build_py': custom_cmd(
+        build_py,
+        [build_std_names, compile_pyke_rules, copy_copyright]),
+    'std_names':
+        custom_cmd(BaseCommand, [build_std_names],
+                   help_doc="generate CF standard name module"),
+    'pyke_rules':
+        custom_cmd(BaseCommand, [compile_pyke_rules],
+                   help_doc="compile CF-NetCDF loader rules"),
+    'clean_source': CleanSource,
+    }
+
+
 setup(
     name='Iris',
     version=extract_version(),
     url='http://scitools.org.uk/iris/',
     author='UK Met Office',
-
+    author_email='scitools-iris-dev@googlegroups.com',
     packages=find_package_tree('lib/iris', 'iris'),
     package_dir={'': 'lib'},
-    package_data={
-        'iris': list(file_walk_relative('lib/iris/etc', remove='lib/iris/')) + \
-                list(file_walk_relative('lib/iris/tests/results',
```

Comment:
Note that: before, we just put everything in lib/iris/tests/results. Now we are being explicit with a glob in MANIFEST. Not necessarily better or wrong - just different.